### PR TITLE
MWPW-126022 [Milo Block] Rich Text

### PR DESCRIPTION
**Pull request by sharmrj**
https://jira.corp.adobe.com/browse/MWPW-126022

please note that other css concerned with rich text will be handled in https://jira.corp.adobe.com/browse/MWPW-129619

[docx file](https://adobe.sharepoint.com/:w:/r/sites/adobecom/_layouts/15/Doc.aspx?sourcedoc=%7BC26B49A9-4DFF-4FDD-8993-CFA7F156E805%7D&file=Document.docx&action=default&mobileredirect=true)
test url: https://richtext--helpx-internal--adobecom.hlx.page/content/help/en/internal/note-test

### CODE REVIEW

**Comment by adeshmukh-1585** [comment]
Hi @sharmrj ,

We have handled the cases when this component is inside step and procedure or accordion ?

**Review by adeshmukh-1585 — DISMISSED** [review]

File: styles/styles.css [hunk 1]
```diff
@@ -155,7 +155,7 @@ footer {
 .layout-container {
   max-width: 1200px;
   margin: 0 auto;
-  display: flex;
+  /* display: flex; */
```
Comment (nkthakur48) [inline styles/styles.css]: You could remove commented code

**Review by nkthakur48 — DISMISSED** [review]
Minor comment!

**Comment by sharmrj** [comment]
> Hi @sharmrj ,
> 
> We have handled the cases when this component is inside step and procedure or accordion ?

Hi @adeshmukh-1585 

Franklin/Milo don't allow block nesting. Since both procedure/step and accordion are their own block, we've no straightforward way of nesting a note block inside procedure/step or accordion. Other rich text features should be available inside all other blocks as a matter of course (since word is a rich text editor and Franklin should transfer all text styles (font size, italics, bold, and so on) to the final html document.

**Review by nkthakur48 — APPROVED** [review]

**Review by dagahimanshu — APPROVED** [review]

**Review by praveenrv — APPROVED** [review]

**Pull request merged**